quicksend.py

```diff
@@ -0,0 +1,23 @@
+import socket
+import signal
+import sys
+import time
+
+HOST='127.0.0.1'
+PORT=5000
+
+def main():
+	s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+	s.connect((HOST, PORT))
+	
+	print(f"Client is connected {s}")
+	
+	binary_int_to_send = int(sys.argv[1], 2)
```

Comment:
Takes the first argument in the command line `python3 quicksend.py <binary, this argument>` and sends it to the socket.